projects/optic/src/commands/diff/diff.ts

```diff
@@ -177,11 +181,14 @@ const runDiff = async (
       logger.info(log);
     }
 
-    if (!config.isInCi) {
-      logger.info('');
-      logger.info(
-        `Configure check standards in optic cloud or your local optic.dev.yml file.`
-      );
+    logger.info('')
+
+    if (!hasOpticUrl) {
+      logger.info(chalk.blue.bold(`See the full history of this API by running "optic add ${path.parse(baseFile.sourcemap.rootFilePath).base}"`))
```

Comment:
You won't actually add the full API history unless you add `optic add spec --history-depth 0`